### PR TITLE
Authorization

### PR DESCRIPTION
**Pull request by mutinhiri**
[Video Presentation](https://www.loom.com/share/693a07246eec4e0e875e85d7b1491ca7)
In this Pr i created a Ruby on Rails application that allows the user to:

-register and log in, so that the data is private to them.
-introduce new transactions associated with a category.
-see the money spent on each category.

### CODE REVIEW

File: app/views/groups/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <%= form.text_field :name, autofocus: true, class: "input is-medium", placeholder: "Group or Company name" %>
+  </div>
+
+  <div class="field">
+    <%= form.text_field :icon, class: "input is-medium", placeholder: "URL address for icon" %>
+  </div>
+
+  <div class="actions">
+    <%= form.submit "Create Group", class: "button is-success is-large is-fullwidth" %>
+  </div>
```
Comment (DeliceLydia) [inline app/views/groups/_form.html.erb]: - [x] Please Follow the [given design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) that includes colors etc.. try to use only colors used in the design

File: app/views/groups/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<%= render "form", group: @group %>
+<br>
+<div class="navbar is-fixed-bottom bottom">
+  <%= link_to "Back to Groups", groups_path, {method: :get, class: "button is-large is-outlined is-fullwidth bottom-btn"} %>
+</div>
```
Comment (DeliceLydia) [inline app/views/groups/new.html.erb]: - [x] Even the navbar should use only colors used in the design please address that

File: app/views/investments/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  <div class="field">
+    <div class="select is-multiple is-large is-fullwidth">
+      <%= form.collection_select(:group_id, Group.all, :id, :name, {}, {:multiple => true}) %>
+    </div>
+  </div>
+
+  <div class="actions">
+    <%= form.submit "Create Investment", class: "button is-success is-large is-fullwidth" %>
+  </div>
```
Comment (DeliceLydia) [inline app/views/investments/_form.html.erb]: - [x] Same as here please use colors used in the design

File: app/views/investments/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        <div class="column is-one-fifth">
+          <%= button_to "Delete", group_investment_path(id: investment.id), {method: :delete, class: "button is-danger is-medium"} %>
+        </div>
+      </div>
+    </div>
+  <% end %>
+</div>
+<div class="navbar is-fixed-bottom bottom">
+  <%= link_to "New Investment", new_group_investment_path, {method: :get, class: "button is-large is-outlined is-fullwidth bottom-btn"} %>
+</div>
```
Comment (DeliceLydia) [inline app/views/investments/index.html.erb]: - [x] Please use colors that were used in the [given design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)

File: README.md [hunk 5]
```diff
@@ -1,24 +1,127 @@
...
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+
+Give a ⭐️ if you like this project! 😄
+
+
+## Acknowledgments
```
Comment (DeliceLydia) [inline README.md:127]: - [x] Please give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @mutinhiri, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Major highlights for your project:
- Awesome video presentation
- App is working as expected
- Followed git worflows


## Required Changes ♻️
_Check the comments under the review._




### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by DanSam5K — APPROVED** [review]
## Hi @mutinhiri ✌️ Approved ✌️ ,
- [x] Good job implementing the changes suggested ✔️ 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**